src/backend/assets/js/session.js

```diff
@@ -1,20 +1,21 @@
 $(document).ready(function() {
+  // name is variable defined in the script in the session template file
+  var eventName = name;
```

Comment:
Hmm. Thanks, is it good to keep name in the window then. Not sure, how can we share data between the js file with .hbs  without this.
But can we do it like `var a = {};
                                     a.name = {{eventurls.name}}

`

Now, `a` will be in window and not name. Than we access a.name in the other file.